pages/api/studentPortal/quizzes/FirstLang.ts

```diff
@@ -0,0 +1,67 @@
+type QuizData = {
```

Comment:
rename this file to firstProgrammingLanguage.ts only components and pages are capitalized